serveradmin/backup.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import zipfile
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+from ideasbox import __version__
+
+
+def make_name():
+    """Return backup formatted file name."""
+    return '{}.zip'.format('_'.join([
+        settings.IDEASBOX_ID,
+        __version__,
+        datetime.now().strftime(Backup.DATE_FORMAT)
+    ]))
+
+
+class Backup(object):
+
+    FORMAT = 'zip'
+    DATE_FORMAT = "%Y%m%d%H%M"
```

Comment:
I've thought about it since then, because I think your readability point is a real one. But then I ended to the conclusion that those file names are not made to be *read* by humans, but made to be *readable* by humans, and at the end, as soon as they are *readable*, the important point is robustness across many unknown workflows.

So, as for now, I'm going this way. We still have one month before release, so we have a bit of time for a better idea to rise up :)